servicebroker/test/routes/binding_test.js

```diff
@@ -113,6 +113,23 @@ describe('binding RESTful API', function() {
             })
           });
       });
+      context('when there is no policy in request', function() {
+        it("return a 201", function(done) {
```

Comment:
Question: if we do not provide a policy here, where is the service supposed to pick up the policy from to apply to the application? We should support defaulting policy defaulting before we enable this.